documentation/MIGRATION guide for bumping up to v30.0.0.md

```diff
@@ -0,0 +1,196 @@
+## Guide for bumping Javascript/Typescript SDK to version 30.0.0+
+
+This document is intended to help users bump Javascript/Typescript SDK version to version 30.0.0+ as there are some significant changes since version 30.0.0.
+
+**For users new to the Javascript/Typescript SDK ([azure-sdk-for-js](https://github.com/Azure/azure-sdk-for-js)) please see [how to use](./how-to-use.md).**
+
+## Why switch to version 30.0.0+?
+
+Compared to libraries under version 30.0.0, the newer version has the following changes: 
+
+1. Authentication: The packages `@azure/ms-rest-nodeauth` or `@azure/ms-rest-browserauth` are no longer supported. Use package [@azure/identity](https://www.npmjs.com/package/@azure/identity) instead. Select a credential from Azure Identity examples based on the authentication method of your choice
+1. Callbacks: Method overloads that used callbacks have been removed and the use of promises is encouraged instead
+1. List operations now return an iterable result that follows the `PagedAsyncIterableIterator` interface as opposed to the previous model where you had to make a new request using the link to the next page.
+1. Long running operations i.e. the Lro related object returned by methods whose names started with `begin`, now uses `pollUntilDone` to check whether the request is finished, instead of `pollUntilFinished`. To get the final result, use the corresponding method that will have the suffix `AndWait`.
+1. The SDK only supports ECMAScript 2015 (ES6) and beyond, all projects that referenced this SDK should be upgraded to use ES6.
+
+If you have an existing application that uses the Javascript/Typescript Azure SDK packages and you're interested in updating your application to use the newer JavaScript/Typescript Azure SDK packages, then the good news is that there is very little for you to do. Here's the things that have changed with this new set of SDKs:
+
+## Current status
+Currently, we have previewed several packages to version 30.0.0+. See the below list and have a try.  
+1. [@azure/arm-resources](https://www.npmjs.com/package/@azure/arm-resources/v/30.0.0-beta.1)
+1. [@azure/arm-features](https://www.npmjs.com/package/@azure/arm-features/v/30.0.0-beta.2)
```

Comment:
i think we can include some examples but no need to maintain the full list, this will require us to update this doc constantly.

Reply:
changed.